File: src/JavaScript/easy/math/problem1716.js
/**
 * @param {number} n
 * @return {number}
 */

var totalMoney = function(n) {
    let sum = 0
    let money = 0
    let week = 0
    for(let i = 1; i <= n; i++){
        money++
        sum += money
        if(i % 7 == 0) {
            week++
            money = week
        }
        
    }
    return sum
};

// let n = 4
// let n = 10
let n = 20

totalMoney(n)

File: src/Python/easy/math/problem1716.py
class Solution:
    def totalMoney(self, n: int) -> int:
        sum = 0
        money = 0
        week = 0
        for i in range(1,n + 1):
           money += 1
           sum += money
           if i % 7 == 0:
                week += 1
                money = week
        return sum


n = 10

solution = Solution()
solution.totalMoney(n)